pkgs/development/libraries/gtk+/3.x.nix

```diff
@@ -36,12 +37,12 @@ stdenv.mkDerivation rec {
     })
   ];
 
-  buildInputs = [ libxkbcommon epoxy json-glib ];
+  buildInputs = [ libxkbcommon epoxy json-glib ]
+    ++ optionals stdenv.isDarwin [ AppKit Cocoa ];
```

Comment:
Do you need this with x11? We might want something like ```withQuartz``` or similar to provide here.

Reply:
This should probably be propagated here. Looks like it's causing newly failing:

https://hydra.nixos.org/eval/1474251#tabs-now-fail

Reply:
My bad, I forgot that didn't test the dependent packages yet. Because frameworks are impure they shouldn't be propagated unless there's a good reason to do so. Which is only `Cocoa` in this case. https://github.com/NixOS/nixpkgs/compare/90b991625b45...ac95ddb008a0